docs/nuget-org/nuget-org-faq.yml

```diff
@@ -263,3 +263,29 @@ sections:
           2. Click on this url: [https://www.nuget.org/account/delete](https://www.nuget.org/account/delete) and follow the steps to submit the request for deleting the account.
           
           Our customer support will process this request and perform the account deletion.
+          
+      - question: Error connecting to nuget.org or restoring packages from `https://api.nuget.org/v3/index.json`?
+        answer: |
+          If you are experiencing connectivity issues while connecting to nuget.org or restoring packages from https://api.nuget.org/v3/index.json, such as:
+
+          > "The SSL connection could not be established"
+
+          , or
+
+          > “[nuget.org] Unable to load the service index for source https://api.nuget.org/v3/index.ison. An error occurred while sending the request.
+          The underlying connection was closed: An unexpected error occurred on a send. Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host.
+          An existing connection was forcibly closed by the remote host.”
+
+          , or
+
+          > “This page can’t be displayed. Turn on TLS 1.0, TLS 1.1, and TLS 1.2 in Advanced settings and try connecting to https://api.nuget.org again. If this error persists, it is possible that this site uses an unsupported protocol or cipher suite such as RC4 (link for the details), which is not considered secure. Please contact your site administrator.”
+
+          , this could be related to [the changes to nuget.org supported cipher suites](https://github.com/NuGet/Announcements/issues/60). 
+
+          Please ensure that TLS 1.2 and any of the following cipher suites have been enabled on your machine:
```

Comment:
>the following cipher suites [](http://example.com/codeflow?start=48&length=27)

I am a little concerned about listing the cipher suites here:
1. This list of cipher suites is now used by one of the two CDN providers we have, and the two CDN providers support different sets of cipher suites today. We run the active failover in different regions to ensure the continuous operation of the service, so there is no guarantee that applying these cipher suites work for customers (especially the latter two for legacy systems). Though we know that what we have at the two CDN providers support the known systems for customers.
2. For the CDN provider who is using this list of cipher suites, they plan to switch to a different set by the end of this year, so it will be changed soon. Once they finish the work on their side, we have the chance to align the cipher suites used at the two CDN provider early next year. If we'd like to show what cipher suites our endpoint supports in the document, it may be better to do that until we align the two CDN providers next year.
3. Checking SSL Labs helps but it doesn't mean that the result is correct. Because the node that SSL Labs runs the test may locate in a different region from the customer, and in that location, it may be a different CDN provider with a different set of cipher suites.